data-host/chrome-extension/src/pages/Popup/index.tsx

```diff
@@ -1,16 +1,9 @@
 
-import { Store } from 'webext-redux';
-
-import { REDUX_PORT_NAME } from '../../redux/port';
-
 import { FractalUI } from '@trustfractal/data-host';
 import './index.css';
 
-const store = new Store({ portName: REDUX_PORT_NAME });
+let ui = new FractalUI();
+ui.render(document.getElementById('app-container'));
 
-store.ready().then(async () => {
-    let ui = new FractalUI();
-    ui.render(document.getElementById('app-container'));
-});
 
 if (module.hot) module.hot.accept();
```

Comment:
We may need to reinitialize something for hot reloading to work? But NBD really.

Reply:
It is currently working.